experiments/optimizer/README.md

```diff
@@ -89,25 +90,25 @@ Let's investigate that violation.
 Imagine the results of this violation in a public environment - _/top/secret/data_ might have been publicly available
 if the Publish tier was not fully secured.
 
-## Test #3: Run the DOT Maven Plug-In on your project
+## Test #3: Run the DOT Maven Plugin on your project
 
 - Find a Dispatcher configuration that you have for your own AEM environment
 - Setup a POM file:
-  - If there is **no** pom.xml for that configuration:
-    - Copy `aem-dispatcher-experiments/pom.xml` to a folder whose children eventually contain both 
+  - If there is **no** pom.xml file in the `dispatcher` module of your project:
+    - Copy `dispatcher-config-basic/pom.xml` to a folder whose children eventually contain both 
       `dispatcher.any` and `httpd.conf` files.
-    - Feel free to edit the copied pom's <artifactId> to make it your own.
-  - If there **is** an existing POM.xml file:
-    - Add the `dispatcher-optimizer-maven-plugin` to the _build/plugins_ section of that existing POM.
-    - See `aem-dispatcher-experiments/pom.xml` for an example.
-    - Update the `dispatcherModuleDir` and `dispatcherConfigPath` to point to the exact file as required.
+    - Feel free to edit the copied pom.xml's project details (`<groupId>`, `<artifactId>`, etc.) to make it your own.
+  - If there **is** an existing pom.xml file in the `dispatcher` module of your project:
+    - Add the `dispatcher-optimizer-maven-plugin` plugin details to the _build/plugins_ section of your existing POM.
+    - See `dispatcher-config-basic/pom.xml` for an example.
 - `cd` to the directory with that POM file.
 - Execute `mvn dispatcher-optimizer:analyze`
-- Watch the console for information about the configuration (in some cases these are directed to the logs).
+- Watch the console for information about the configuration (in some cases, additional details about the detected 
+violations is printed to the logs).
```

Comment:
I think the redirection is distracting, and I want to emphasize that they look at the log entries, not just the violations.  How about:
```
- Examine the resulting output in the console.  In addition to violations, there will be information
  about surprises that were encountered while parsing the configuration files.  Those are worth reviewing, especially
  errors and warnings.
```
Also on line 61 (ish) above...